README.md

```diff
@@ -158,6 +158,17 @@ Generate manifests e.g. CRD, RBAC YAML files etc.
 
 To develop/debug kruise controller manager locally, please check the [debug guide](./docs/debug/README.md).
 
+## Uninstall
+
+To uninstall kruise from a Kubernetes cluster:
+
+```bash
+export KUBECONFIG=PATH_TO_CONFIG
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/kruiseio/kruise/master/scripts/uninstall.sh)"
+```
+
+Note that this will lead to all resources created by Kruise to be deleted.
```

Comment:
This is a dangerous operation. Let us make it clear. How about

Note that this will lead to all resources created by Kruise, including webhook configurations, services, namespace, CRDs, CR instances and Pods managed by Kruise controller, to be deleted! Please do this **ONLY** when you fully understand the consequence.

Reply:
done